topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -203,9 +203,13 @@ To proceed from here it is expected that:
 6. In your inventory file, you have written the full DNS hostname that has been provided, and **not** `localhost`, as we will be requesting SSL certificates.
 
 
-> ### {% icon tip %} Ubuntu or Debian, CentOS or RHEL?
-> The training tutorials are designed to work on either Ubuntu or CentOS, and the roles we use are definitely compatible with both. If any of the variable values differ between Ubuntu and CentOS, we try to note it in the tutorial. Any places we don't note it are bugs.
-{: .tip}
+> ### {% icon comment %} Ubuntu or Debian, CentOS or RHEL?
+> The training tutorial is only tested on Ubuntu, as that is the training platform most often used. The training *should* work on RHEL/CentOS, and the roles we use are definitely compatible with both. If any of the variable values differ between Ubuntu and CentOS, we try to note it in the tutorial. Any places we don't note it are bugs.
+>
+> There are known issues with CentOS7 and python3, which is used in this tutorial. If you use this setup, you should have python2 and python3 coexisting, and use python2 for Ansible (i.e. do not set an interpreter in your `ansible.cfg`) and python3 for Galaxy. This setup requires numerous changes that you will need to discover, which are not covered in this tutorial.
```

Comment:
```suggestion
> There are known issues with CentOS7 and python3, which is used in this tutorial. If you use this setup, you should have python2 and python3 coexisting, and use python2 for Ansible (i.e. do not set `interpreter_python` in your `ansible.cfg`) and python3 for Galaxy. This setup requires numerous changes that you will need to discover, which are not covered in this tutorial.
```

Reply:
We should also mention that both `python-psycopg2` and `python3-psycopg2` need to be installed, not sure if here or in the PostgreSQL section.